docs/anomaly-detection/QuickStart.md

```diff
@@ -121,63 +121,57 @@ Below are the steps to get `vmanomaly` up and running inside a Docker container:
-
-```sh
-export YOUR_LICENSE_FILE_PATH=path/to/license/file
-export YOUR_CONFIG_FILE_PATH=path/to/config/file
-docker run -it -v $YOUR_LICENSE_FILE_PATH:/license \
-               -v $YOUR_CONFIG_FILE_PATH:/config.yml \
-               vmanomaly /config.yml \
-               --licenseFile=/license \
-               --loggerLevel=INFO \
-               --watch
-```
+3. Create and modify your `config.yml` file to your liking. An example can be found [here](https://docs.victoriametrics.com/anomaly-detection/quickstart/#example)
 
-In case you found `PermissionError: [Errno 13] Permission denied:` in `vmanomaly` logs, set user/user group to 1000 in the run command above / in a docker-compose file:
+4. Start the `vmanomaly` Docker container with a *license file*, use the command below.
+**Make sure to replace `YOUR_LICENSE_FILE_PATH`, and `YOUR_CONFIG_FILE_PATH` with your specific details**:
 
 ```sh
-export YOUR_LICENSE_FILE_PATH=path/to/license/file
-export YOUR_CONFIG_FILE_PATH=path/to/config/file
-docker run -it --user 1000:1000 \
-               -v $YOUR_LICENSE_FILE_PATH:/license \
-               -v $YOUR_CONFIG_FILE_PATH:/config.yml \
-               vmanomaly /config.yml \
+docker run -it \
+               -v ./license:/license \
+               -v ./config.yml:/config.yml \
+#              -v ./data:/tmp/vmanomaly \ # Enable if you are using settings.restore_state: True
+               victoriametrics/vmanomaly:v1.28.0 /config.yml \
+               --port 8490:8490 \
                --licenseFile=/license \
                --loggerLevel=INFO \
                --watch
 ```
 
 ```yaml
-# docker-compose file
+# docker-compose.yml file
 services:
-  # ...
   vmanomaly:
-    image: victoriametrics/vmanomaly:v1.27.1
+    image: victoriametrics/vmanomaly:v1.28.0
     volumes:
-        $YOUR_LICENSE_FILE_PATH:/license
-        $YOUR_CONFIG_FILE_PATH:/config.yml
+      - ./license:/license
+      - ./config.yml:/config.yml
+      # - ./data:/tmp/vmanomaly # Enable if you are using settings.restore_state: True
     command:
       - "/config.yml"
       - "--licenseFile=/license"
       - "--loggerLevel=INFO"
       - "--watch"
-    # ...
+    ports:
+      - "8490:8490"
+    # environment: # Enable if you are using settings.restore_state: True
+    #   - "VMANOMALY_MODEL_DUMPS_DIR=/tmp/vmanomaly/models"
+    #   - "VMANOMALY_DATA_DUMPS_DIR=/tmp/vmanomaly/data"
 ```
 
-For a complete docker-compose example please refer to [our alerting guide](https://docs.victoriametrics.com/anomaly-detection/guides/guide-vmanomaly-vmalert/), chapter [docker-compose](https://docs.victoriametrics.com/anomaly-detection/guides/guide-vmanomaly-vmalert/#docker-compose)
-
+> In case of `PermissionError: [Errno 13] Permission denied:` in `vmanomaly` logs {{% deprecated_from "v1.28.0" anomaly %}}, set user/user group to 1000, e.g. `--user 1000:1000` in the `docker run` command or `user: "1000:1000"` in the `docker-compose.yml` file.
```

Comment:
Because the userid is already set to 1000 in the dockerfile this does nothing and should be removed.